misp_stix_converter/stix-to-misp.py

```diff
@@ -1,69 +1,94 @@
 #!/usr/bin/env python3
-
-##########################
-#     WRAPPER SCRIPT     #
-#     DEALS WITH STIX    #
-#     TO MISP            #
-##########################
-
+"""Wrapper script providing STIX to MISP conversion. """
```

Comment:
Why did you change this?

It's consistent throughout the project.

Reply:
Ok, interesting, newbie from planet python - thought that one liner would convey more meaning and suit pep8